modules/kubernetes-addons/argocd/variables.tf

```diff
@@ -44,3 +44,18 @@ variable "add_on_config" {
   default     = {}
   description = "Configuration for managing add-ons via ArgoCD"
 }
+
+variable "addon_context" {
+  type = object({
+    aws_caller_identity_account_id = string
+    aws_caller_identity_arn        = string
+    aws_eks_cluster_endpoint       = string
+    aws_partition_partition        = string
+    aws_region_id                  = string
+    aws_region_name                = string
```

Comment:
@Zvikan Could you please remove `aws_region_id` and just use `aws_region_name` everywhere. We dont need both which gives the sameoutput